modules/custom/az_migration/migrations/az_files_oembed.yml

```diff
@@ -0,0 +1,33 @@
+id: az_files_oembed
+label: Remote Files (oEmbed)
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+source:
+  plugin: d7_file
```

Comment:
I don't think we even need to import these as files at all, and instead import them as media entities.